app/views/devise/passwords/edit.html.erb

```diff
@@ -23,7 +23,7 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Change my password" %>
+    <%= f.submit "Change my password"  class: "btn btn-primary" %>
```

Comment:
please add ',' it should be ` <%= f.submit "Change my password",  class: "btn btn-primary" %>`